search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityOperationsHandler.kt

```diff
@@ -26,8 +31,11 @@ class TemporalEntityOperationsHandler(
     @PostMapping("/query", consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
     suspend fun queryEntities(
         @RequestHeader httpHeaders: HttpHeaders,
+        @RequestParam params: MultiValueMap<String, String>,
```

Comment:
when doing a query with a POST, parameters will not anymore be in the URL but in the request body (see 6.24 in the specification for details about this). So you have to use the `params` object that is retrieved from the request body (at line 42 below).

Reply:
refactor and added in query service. But test is getting initialization failed.